install.yml

```diff
@@ -65,18 +65,26 @@
     - name: set correct file permission for everything in the deployment
       file:
         path: "{{ root_path }}"
+        group: "{{ ngi_sw_group }}"
+        state: directory
+        recurse: yes
+        mode: g+rwXs,o=rX
```

Comment:
Don't we want to remove all permissions for others? (o-rwx)

Reply:
no, others need to be able to access stuff under this hierarchy in order to use the deployment